tests/integration/test_airgapped.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import logging
+import uuid
+
+import pytest
+import requests
+from fixtures import deploy_package
+from helpers import (
+    ACTIVE_STATUS,
+    APP_NAME,
+    PRO_AIRGAPPED_SERVER_CHANNEL,
+    PRO_AIRGAPPED_SERVER_ENDPOINT,
+    PRO_AIRGAPPED_SERVER_NAME,
+    ensure_model,
+)
+from pytest_operator.plugin import OpsTest
```

Comment:
nit: there is a small conflation of imports:
```suggestion
from pytest_operator.plugin import OpsTest

from fixtures import deploy_package
from helpers import (
    ACTIVE_STATUS,
    APP_NAME,
    PRO_AIRGAPPED_SERVER_CHANNEL,
    PRO_AIRGAPPED_SERVER_ENDPOINT,
    PRO_AIRGAPPED_SERVER_NAME,
    ensure_model,
)
```
This one looks more precise 🙂

Reply:
That's how the linter wants it to be.